content/6_Advanced/Treaps.mdx

```diff
@@ -197,8 +197,7 @@ pt calc(pt x) {
 }
 	
 pair<pt, pt> split(pt t, int v) { // >= v goes to the right
-	if (!t)
-		return {t,t};
+	if (!t) return {t,t};
```

Comment:
bruh

Reply:
~~add braces~~

Reply:
don't u dare

Reply:
at least put a space after the comma like a normal person

Reply:
but this isn't even my code...

Reply:
you can still fix it